src/ast/passes/printer.cpp

```diff
@@ -364,15 +358,20 @@ void Printer::visit(For &for_loop)
   }
 
   out_ << indent << " decl\n";
-  print(for_loop.decl);
+  ++depth_;
```

Comment:
Why were these additional depth inc/dec added?

Reply:
The output is the same.